apps/public-docsite-resources/just.config.ts

```diff
@@ -6,5 +6,5 @@ preset();
 task('generate-json', () => generatePageJsonFiles(require('./config/api-docs')));
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26872&commitId=4480f52d2be1021ec0f14a993ca97ad9d7d82486&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Divider.Divider Vertical Example Shorthand.chromium.png|338| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>